src/directives/routeHref.js

```diff
@@ -0,0 +1,39 @@
+angular.module('bicker_router').directive('routeHref', (Route, $location, $timeout) =>
+  ({
+    restrict: 'A',
+    scope: true,
+
+    link(scope, iElement, iAttrs) {
+
+      if (iAttrs.ignoreHref === undefined) {
+        iElement.click(function(event) {
+          event.preventDefault();
+          let url = iElement.attr('href');
+
+          if (!Route.isHtml5ModeEnabled()) {
+            url = url.replace(/^#/, '');
+          }
+          
```

Comment:
There's a heap of return statements that have been introduced due to coffee's implied return, that don't actually make sense. I'd like to see all of those fixed up lest it confuse people down the line. This one's relatively innocuous, but it's the first one I've seen so I'm commenting here.